Configurateur/front/lib/screens/landing-page/landing_page.dart

```diff
@@ -192,196 +46,138 @@ class LandingPageState extends State<LandingPage> {
+                                    offset: Offset(0.75, 0.75),
+                                    blurRadius: 1.5,
+                                  ),
+                                ],
+                              ),
+                            ),
+                            Padding(
+                              padding: const EdgeInsets.only(
+                                  top:
+                                      15), // Espacement inférieur pour le texte
+                              child: Text(
+                                'Des conteneurs disponibles partout en france !',
+                                style: TextStyle(
+                                  fontSize: 20,
+                                  color:
+                                      Provider.of<ThemeService>(context).isDark
+                                          ? darkTheme.secondaryHeaderColor
+                                          : lightTheme.secondaryHeaderColor,
+                                ),
+                              ),
                             ),
-                            textStyle: TextStyle(
-                                // fontSize: 13.0
-                                )),
-                        child: Text(
-                          'En savoir plus',
-                          style: TextStyle(
-                            color: Provider.of<ThemeService>(context).isDark
-                                ? darkTheme.primaryColor
-                                : lightTheme.primaryColor,
-                          ),
-                        ),
+                          ]),
+                      const SizedBox(width: 100),
+                      Image.asset(
+                        'assets/iphonenew.png',
                       ),
-                    ),
-                    const SizedBox(width: 20),
-                    Padding(
-                      padding: const EdgeInsets.only(top: 35),
-                      child: ElevatedButton(
-                        style: ElevatedButton.styleFrom(
-                          padding: const EdgeInsets.symmetric(
-                              horizontal: 25, vertical: 15),
-                          shape: RoundedRectangleBorder(
-                            borderRadius: BorderRadius.circular(
-                                20.0), // Définit le rayon du bouton arrondi
-                          ),
-                        ),
-                        onPressed: () => goToCreation(),
-                        child: Text(
-                          'Créer mon conteneur',
-                          style: TextStyle(
-                            color: Provider.of<ThemeService>(context).isDark
-                                ? darkTheme.primaryColor
-                                : lightTheme.primaryColor,
-                          ),
-                        ),
+                    ],
+                  )),
```

Comment:
Maybe add , after the first ) so that the file is more readable

Reply:
done